noisy-simulator/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "noisy-simulator"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+nalgebra = { workspace = true }
+num-complex = { workspace = true }
+rand = { workspace = true }
+rayon = { workspace = true }
```

Comment:
From rayon's GitHub repo:

> "By default, when building to WebAssembly, Rayon will treat it as any other platform without multithreading support and will fall back to sequential iteration. This allows existing code to compile and run successfully with no changes necessary, but it will run slower as it will only use a single CPU core."